s3storageprovider/src/main/java/org/duracloud/s3storage/S3StorageProvider.java

```diff
@@ -75,16 +75,16 @@
     private final Logger log = LoggerFactory.getLogger(S3StorageProvider.class);
 
     protected static final int MAX_ITEM_COUNT = 1000;
-    private static final StorageClass DEFAULT_STORAGE_CLASS =
+    protected static final StorageClass DEFAULT_STORAGE_CLASS =
         StorageClass.Standard;
 
     private static final String UTF_8 = StandardCharsets.UTF_8.name();
-    protected static final String HIDDEN_SPACE_PREFIX = "hidden-";
+    public static final String HIDDEN_SPACE_PREFIX = "hidden-";
```

Comment:
This is so that `SwiftStorageProviderTest` has access to it. I can change it back to protected and set a separate static variable in the class itself if preferred.

Reply:
Ah, I missed that reference. This is fine, I'd rather have it be public than duplicated.